docs/Report.md

```diff
@@ -1399,13 +1399,86 @@ Provide all the info and we are good to go. We will then be asked to provide the
+
+Next, when we are back in the OwnerController menu, we can hit 4 in order to add a new book. This is a long process... You need to first add, or pick an existing publisher, then create or add authors, then create the actual book. The first thing you see when you enter this menu is a list of publishers, and you will be asked if your publisher is listed. If it is, you will be asked for their ID, and the publisher's ID will be assigned to that book. If it is not, you will be prompted to create a new publisher.
+
+![publisher_menu](./photos/publisher_menu.png)
+
+For the sake of demoing, I will show the flow for wehen a publisher was not listed. One thing to note in the below screenshot, is that the user is prompted to keep entering phone numbers until they say they are done. This is because publishers can have as many phone numbers as they want.
+
+![creating_publisher](./photos/creating_publisher.png)
+
+Next, you are prompted to create the book. Note that error checking is occuring to ensure that ISBN and title are unique, and you will be prompted to enter a different value if it is not unique.
+
+![adding_book](./photos/adding_book.png)
+
+Next, you will be prompted to add an author. This is the same as adding a publisher. You see all the existing ones, and if the author already exists, you will add select their ID, and if they don't, then you will be prompted to create a new one. I will select a known author.
+
+![select_author](./photos/select_author.png)
+
+Next, since a book can have any number of authors, you will be asked if you want another. In this case, you will be prompted to either enter an existing ID, or create a new author. Once you are done adding authors, you can select 2, which is no, and then you will see the book is successfully added, you will be prompted to hit enter to continue, then you will be returned back to the owner menu.
+
+![select_author](./photos/add_another_author.png)
+
+The next thing we can do as an owner, is remove a book. This workflow is relatively straightforward. You are given a list of books, and you are asked which book you would like to remove. 
+
+![owner_remove_book](./photos/owner_remove_book.png)
+
+If you enter an invalid ISBN, that will be captured. Once you enter a valid ISBN, you get a message that it has been removed, and you are prompted to hit enter to continue.
+
+![removing_book](./photos/removing_book.png)
+
+The last functionality of the owner is to generate reports, which you do by hitting 6 in the owner menu. Once you do, you will see the report menu:
+
+![report_menu](./photos/report_menu.png)
+
+You have the option to see Sales VS Cost, Sales by Author, and Sales by Author. I will demonstrate the Sales by Author report, which you can select by entering 4. As explained above, we did not implement a date capturing feature correctly, so we have to unfortunately prompt the user for the current date, which is stored by the database for when it creates a new Report row.
```

Comment:
'Sales by author and Sales by author', one of these should be something else I assume.